vendor/modules.txt

```diff
@@ -11,26 +11,26 @@ github.com/code-ready/clicumber/testsuite
 github.com/code-ready/clicumber/util
 # github.com/code-ready/goodhosts v0.0.0-20190712111040-f090f3f77c26
 github.com/code-ready/goodhosts
-# github.com/code-ready/machine v0.0.0-20190731093717-b6d974ad44d0
+# github.com/code-ready/machine v0.0.0-20190731093717-b6d974ad44d0 => ../machine
```

Comment:
????

Reply:
ah this might be a side effect of how I was doing testing (`replace ( github.com/code-ready/machine => ../machine` ) in go.mod